### PR TITLE
chore(js): Bump emotion 11.{1,3} -> 11.9

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -179,6 +178,7 @@
     "@storybook/theming": "6.5.9",
     "@types/node": "^18.0.0",
     "@visual-snapshot/jest": "6.0.2",
+    "assert": "^2.0.0",
```
Comment (evanpurkhiser) [inline package.json:181]: I put this in. the wrong place in my #36399 PR

**Review by malwilley — APPROVED** [review]

**Review by scttcper — APPROVED** [review]

**Review by dashed — APPROVED** [review]
wagmi

**Comment by priscilawebdev** [comment]
tests are failing :/

**Comment by evanpurkhiser** [comment]
Uhg, will deal with it later

**Pull request closed without merging**